### PR TITLE
Feature: add isRefreshing and RefreshComponent props

### LINKED ISSUE #108: Refershing/Refresh Control
Any possibilities of add this ?

### PR DESCRIPTION
**Pull request by pdpino**
(Partially) fixes #108 

I added `isRefreshing` and `RefreshComponent` props, lmk what u think about this feature

### CODE REVIEW

**Comment by dandresfg** [comment]
I'm taking a look to those changes and this is only an activity indicator. I mean scroll up at the top. Just like to refresh a page in chrome.

**Comment by pdpino** [comment]
@dandresfg I will answer in the issue, to keep the feature discussion there :slightly_smiling_face:

File: src/RefreshSpinner/RefreshSpinner.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+import styles from './RefreshSpinner.styles';
+
+const RefreshSpinner = ({ CustomComponent }) => {
+  if (CustomComponent) {
+    return <CustomComponent style={styles.spinner} />;
+  }
+
+  return <ActivityIndicator size="large" color="blue" style={styles.spinner} />;
```
Comment (hoangnm) [inline src/RefreshSpinner/RefreshSpinner.js:10]: the color here is very specific, should we avoid rendering the default spinner?

Comment (pdpino) [reply src/RefreshSpinner/RefreshSpinner.js:10]: Only allow the custom component? --> would have to be required when `isRefreshing={true}`

Comment (pdpino) [reply src/RefreshSpinner/RefreshSpinner.js:10]: To be honest my solution looks ugly now that I see it :laughing:, more ideas are welcomed

Comment (pdpino) [reply src/RefreshSpinner/RefreshSpinner.js]: I simplified this, see comment below

**Pull request merged**